### PR TITLE
Announce Jetbrains as Silver Sponsor

### PR DESCRIPTION
**Pull request by vipulgupta2048**
Signed-off-by: Vipul Gupta (@vipulgupta2048) <vipulgupta2048@gmail.com>
Solves https://github.com/pythonindia/inpycon2020-tasks/issues/85

### CODE REVIEW

File: content/blog/2020-announcing-jetbrains-silver-sponsor.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
+Title: Announcing Jetbrains as a Silver Sponsor
+Date: 2020-09-21 17:00
+Author: Vipul Gupta (@vipulgupta2048)
+Category: 2020
+Slug: sponsor-announcement-jetbrains
+Summary: We are happy to introduce **Jetbrains** as a Silver Sponsor for PyCon India 2020. 
+
+We are happy to introduce **Jetbrains** as a Silver Sponsor for PyCon India 2020.
+
+[![Jetbrains](https://user-images.githubusercontent.com/3849885/93641290-a6ee8f80-fa19-11ea-853b-2c8d6bf4b162.png)](https://www.jetbrains.com/)
```
Comment (inovizz) [inline content/blog/2020-announcing-jetbrains-silver-sponsor.md:10]:
```suggestion
[![JetBrains](https://in.pycon.org/2020/assets/images/sponsors/jetbrains.png)](https://www.jetbrains.com/)
```

File: content/blog/2020-announcing-jetbrains-silver-sponsor.md [hunk 2]
```diff
@@ -0,0 +1,14 @@
+Title: Announcing Jetbrains as a Silver Sponsor
```
Comment (inovizz) [inline content/blog/2020-announcing-jetbrains-silver-sponsor.md:1]:
```suggestion
Title: Announcing JetBrains as a Silver Sponsor
```

File: content/blog/2020-announcing-jetbrains-silver-sponsor.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+Date: 2020-09-21 17:00
```
Comment (inovizz) [inline content/blog/2020-announcing-jetbrains-silver-sponsor.md:2]:
```suggestion
Date: 2020-09-30 17:00
```

File: content/blog/2020-announcing-jetbrains-silver-sponsor.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+Author: Vipul Gupta (@vipulgupta2048)
+Category: 2020
+Slug: sponsor-announcement-jetbrains
+Summary: We are happy to introduce **Jetbrains** as a Silver Sponsor for PyCon India 2020.
```
Comment (inovizz) [inline content/blog/2020-announcing-jetbrains-silver-sponsor.md:6]:
```suggestion
Summary: We are happy to introduce **JetBrains** as a Silver Sponsor for PyCon India 2020. 
```

File: content/blog/2020-announcing-jetbrains-silver-sponsor.md [hunk 5]
```diff
@@ -0,0 +1,14 @@
+Title: Announcing Jetbrains as a Silver Sponsor
+Date: 2020-09-21 17:00
+Author: Vipul Gupta (@vipulgupta2048)
+Category: 2020
+Slug: sponsor-announcement-jetbrains
+Summary: We are happy to introduce **Jetbrains** as a Silver Sponsor for PyCon India 2020. 
+
+We are happy to introduce **Jetbrains** as a Silver Sponsor for PyCon India 2020.
```
Comment (inovizz) [inline content/blog/2020-announcing-jetbrains-silver-sponsor.md:8]:
```suggestion
We are happy to introduce **JetBrains** as a Silver Sponsor for PyCon India 2020.
```

**Review by vipulgupta2048 — COMMENTED** [review]
Looking good - deploy @palnabarun

**Review by sayanchowdhury — APPROVED** [review]

**Pull request merged**